Packs/PaloAltoNetworks_IoT/Playbooks/playbook-iot-check-servicenow.yml

```diff
@@ -0,0 +1,147 @@
+id: PANW IoT ServiceNow Tickets Check
+version: -1
+name: PANW IoT ServiceNow Tickets Check
+description: This playbook should be used in a "Job" that is recurring to check the
+  ServiceNow tickets status for the Palo Alto Networks IoT alerts / vulnerabilties
+  (previously Zingbox).
+starttaskid: "0"
+tasks:
+  "0":
+    id: "0"
+    taskid: b2784e0e-12c4-4468-863a-210bf630a2fc
+    type: start
+    task:
+      id: b2784e0e-12c4-4468-863a-210bf630a2fc
+      version: -1
+      name: ""
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "1"
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 50,
+          "y": 50
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+  "1":
+    id: "1"
+    taskid: 1db6684d-1ed6-40b8-8ec3-6ad24fc9520c
+    type: regular
+    task:
+      id: 1db6684d-1ed6-40b8-8ec3-6ad24fc9520c
+      version: -1
+      name: iot-security-check-servicenow
+      description: To close the XSOAR incident if the IoT ServiceNow ticket was closed.
```

Comment:
If your checking with serviceNow you should check if its enabled first.

Reply:
added the check